backend/app/services/config_service.py

```
@@ -0,0 +1,135 @@
+
+        cursor.execute(
+            "SELECT preferences_json FROM user_preferences ORDER BY id DESC LIMIT 1"
+        )
+        result = cursor.fetchone()
+
+        db_channel.close()
+
+        if result:
+            preferences_data = json.loads(result[0])
+            return UserPreferences(**preferences_data)
+        else:
```

Ruff (0.12.2)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)